testsuite/tests/translprim/comparison_table.ml.reference

```diff
@@ -232,7 +232,7 @@
                 (apply f (field 0 param) (field 1 param)))
             map =
               (function f l
-                (apply (field 12 (global List!)) (apply uncurry f)
+                (apply (field 14 (global List!)) (apply uncurry f)
```

Comment:
Yes, it is a fix for the testsuite needed by the patch, because it changes the offset of `List.map` in the signature.